docs/sources/docker-hub/repos.md

```diff
@@ -100,8 +80,14 @@ Hub](https://registry.hub.docker.com/plans/) plan.
 
 ## Webhooks
 
```

Comment:
I think we need a short sentence that defines a "webhook". E.g., "A webhook is an HTTP call-back triggered by a specific event." Or something like that.